analysis.sql

```diff
@@ -0,0 +1,24 @@
+CREATE OR REPLACE PROCEDURE analyze_filterability(table_name text)
```

Comment:
would it make sense to provide some basic docs here? just 1 sentence about the intended use and expected result, maybe a link to some other docs. Also a hint that this should be run in `direct` mode, which might be the only non-obvious part here.

BTW, what happens in non-direct access level? if the error message isn't clear, maybe it's worth to catch this and provide some helpful feedback.

Reply:
Added a few comments. Extended docs will be added later to the admin guide.
I checked and the error message is pretty clear for non-direct mode.